ops/ekl/docker-compose.yml

```diff
@@ -7,16 +7,21 @@ services:
       - ${PWD}/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
     ports:
       - "9200:9200"
+    networks:
+      - erlang-ekl-network
```

Comment:
do you really need a new network? i don't think so

Reply:
networks was used to test syslog, and to dump import/export databases from other containers. When i get the curl import/export working i will remove it.

Reply:
ok, i will merge the PR when the curl command is added